src/object.rs

```diff
@@ -46,10 +46,8 @@ unsafe impl RuntimeType for Object {
 }
 
 #[repr(C)]
-pub struct abi_IInspectable {
```

Comment:
I know users don’t interact with this often, but why the name change? It’s nice to know they this is IInspectable so you can read about it in docs

Reply:
Yeah... it seemed weird that Object's ABI representation is called something else entirely. In C# the `IInspectable` is entirely hidden. In C++ it's just called `IInspectable` throughout.

Reply:
I personally wouldn’t be against calling it IInspectable everywhere. I don’t see what you gain by not using the name, but you lose the ability for people to easily search for more info on it if you make it something else